ledger-tool/src/ledger_utils.rs

```diff
@@ -365,7 +365,7 @@ pub fn load_and_process_ledger(
         exit.clone(),
         abs_request_handler,
         process_options.accounts_db_test_hash_calculation,
-        None,
+        starting_snapshot_hashes.map(|x| x.full.0 .0),
```

Comment:
what's with the extra space after `x.full.0`?

Reply:
we have to revisit this when clean/shrink can get rid of single ref zero lamport accounts.